Sources/Amplitude/AMPStorage.m

```diff
@@ -38,6 +38,9 @@ + (NSString *)getAppStorageAmpDir:(NSString *)instanceName {
     NSArray *paths = NSSearchPathForDirectoriesInDomains(NSApplicationSupportDirectory, NSUserDomainMask, YES);
     NSString *path = [paths firstObject];
     NSString *bundleIdentifier = [[NSBundle mainBundle] bundleIdentifier];
+    if (instanceName == NULL || [instanceName length] == 0) {
```

Comment:
Do we need this? the migration happens always after `instance` or `instanceName`, so we will have the default instance or other customize instances name.

Reply:
In my test, the file name was 
`/Users/dante.tam/Library/Developer/CoreSimulator/Devices/00438800-9EF0-409D-A735-D3A9552A16F0/data/Library/Application Support/com.apple.dt.xctest.tool//amplitude_event_storage.txt`

The double slash messes things up. Adding `DEFAULT_INSTANCE` changes the path URL to
`/Users/dante.tam/Library/Developer/CoreSimulator/Devices/00438800-9EF0-409D-A735-D3A9552A16F0/data/Library/Application Support/com.apple.dt.xctest.tool/DEFAULT_INSTANCE/amplitude_event_storage.txt`